### ISSUE TITLE
NFTokenCreateOffer (Create Buy Offer) - NFToken Tester Harness -  The latest ledger sequence XXX is greater than the transaction's LastLedgerSequence (XXX).

### ISSUE THREAD

**Issue by jremi** [XRPLF/xrpl.js#1901]
Hi,

I'm playing around with the NFToken Tester harness and specifically attempting to create a buy offer and I continue to get the following error:

```
The latest ledger sequence XXXXXX is greater than the transaction's LastLedgerSequence (XXXXXX).
```

The XXXXXX is the ledger number near the time I attempt to perform the transaction.  For example 116850

....

1. On XRPL NFT-Devnet account 1 I successfully mint a token.
2. I then successfully create a sell offer for a specific amount of drops for this newly minted token id.  
3. I know its succesful because when i run the query to get offers I am able to see the newly created sell offer for this token.

Now..... let's change our mental context. I'm now using a different XRPL NFT-Devnet account 2. 

1.  I provide the flags as 0 , i provide the same token id that was minted earlier by the account 1, I enter the same amount of drops as requested in the sellers offer.
2. I then attempt to perform a create buy offer which sends this example `transactionBlob`:

```
{
    "TransactionType": "NFTokenCreateOffer",
    "Account": "r9nF5T4yLbf1at1BHgNnEhHARaMX1Jq1Wt",
    "Owner": "rB6ychxz1qjyxoSHum6PiL3aQBdPG98hdk",
    "TokenID": "00000000757125C0C19B47BDE91FA6739E99267FDF10F1E2C9F45DAF00000014",
    "Amount": "850000000",
    "Flags": 0
}
```

Long story short when I await this transaction, it seems to always fail.

I continue to get back the following error:

```
The latest ledger sequence 116850 is greater than the transaction's LastLedgerSequence (116850).
```

What am I missing?

I have also attempted to update to the latest xrpl.js script src that is used inside the test harness.

The test harness is currently set to use `https://unpkg.com/xrpl@2.1.0-beta.1`.

For the heck of it, I tested also with `https://unpkg.com/xrpl@2.1.1`

Either way, I can't get the buy offer to hit the ledger.

Any insights would be greatly appreciated. 

🙏  J

**Comment by jremi** [comment]
After speaking with another dev on the XRP Ledger Developers Discord nfts channel... We determined the issue was the wrong `Flags` value was set. It needed to be set to value of `8` for enabling transfer of the token.

That being said.... I believe the error messaging / functionality should not allow you to create Offers if the token cannot be transferred.
